common/views/components/MediaObjectBase/MediaObjectBase.tsx

```diff
@@ -132,10 +137,37 @@ const MediaObjectBase: FunctionComponent<Props> = ({
         )}
 
         {partNumber && (
-          <PartNumberIndicator number={partNumber} color={color} />
+          <PartNumberIndicator
+            number={partNumber}
+            description={partDescription}
+            color={color}
+          />
         )}
         <TitleWrapper>{title}</TitleWrapper>
-        {DateInfo}
+        {showPlayButton && (
```

Comment:
Instead of passing around props that are specific to the components implementing this component (e.g. `PodcastCard`), and just pass down some `children`?

e.g.
```tsx
Props = { PostTitleChildren: ReactElement, showDateInfo: boolean }
```

And then have:
```ts
const PodcastCard = () => {
<MediaObjectBase showDateInfo={false} PostTitleChildren={/*...*/} />
}
```

We have had it in the past where we overload components with props very specific to certain situations making them quite hard to read, this way we can remove that complexity.

Reply:
Not sure I follow

Reply:
The play button is very specific to the implementing component, in this case a podcast based component. We could keep the `MediaObjectBase` component  generic by opening slots that other components can fill with content specific to their requirements.

e.g. the play button is a requirement of a Podcast card, the requirement of this more abstract component is to be able to be composable.

Reply:
got ya